tests/Functional/Driver/AbstractDriverTest.php

```diff
@@ -49,6 +52,13 @@ public function testReturnsDatabaseNameWithoutDatabaseNameParameter(): void
         );
     }
 
+    public function testProvidesAccessToTheNativeConnection(): void
+    {
+        $nativeConnection = $this->connection->getNativeConnection();
+
+        self::assertTrue(is_object($nativeConnection) || is_resource($nativeConnection));
```

Comment:
Please see if this could be reworked via more specific assertions combined with `self::logicalOr()`. If the test fails, it will yield a more helpful message than "true is not true".

Reply:
Of course, done.